pkg/localize/locales/en/cmd/kafka.en.toml

```
@@ -402,6 +402,13 @@ one = 'Unique ID of the Kafka instance you want to view'
 description = 'Description for the --name flag'
 one = 'Name of the Kafka instance you want to view'
 
+[kafka.describe.flag.bootstrapserver]
+description = 'Description for the --bootstrap-server flag'
+one = 'If specified, only bootstrap server host of the Kafka instance will be displayed'
+
+[kafka.describe.bootstrapserver.not.available]
+one = 'Unable to get bootstrap server host name'
```

```suggestion
one = 'Kafka instance "{{.Name}}" does not have a bootstrap server URL.'
```

usually we give more specific information. It is not that we are unable, it does not exist.

---

@craicoverflow done, please take a look.